orchestrator/nodes/ftp/ftp.html

```diff
@@ -0,0 +1,130 @@
+        <dt class="mandatory">method <span class="property-type">string</span></dt>
+        <dd>Sets which operation should be performed. If <code>PUT</code>, then "filename" will refer to the remote
+            filename (and content will be what should be uploaded). If <code>GET</code>, then "filename" refers to which
+            file should be downloaded and "content" will be the attribute that will hold the file data (in base64 format).</dd>
+        <dt class="mandatory">username <span class="property-type">string</span></dt>
+        <dd>FTP username</dd>
+        <dt class="mandatory">password <span class="property-type">string</span></dt>
+        <dd>FTP password</dd>
+        <dt class="mandatory">filename <span class="property-type">string</span></dt>
+        <dd>Remote filename (which will be created, if method is PUT, or to be downloaded, if method is GET</dd>
+        <dt class="mandatory">file content <span class="property-type">string</span></dt>
+        <dd>File content to be uploaded (if method is PUT) or where file data is stored (if method is GET)</dd>
+    </dl>
+    <h3>Outputs</h3>
+    <dl class="message-properties">
+        <dt>result <span class="property-type">string | object | buffer</span></dt>
+        <dd>The body of the response. The node can be configured to return the body
+             as a string, attempt to parse it as a JSON string or leave it as a
+             binary buffer.</dd>
+        <dt>statusCode <span class="property-type">number</span></dt>
+        <dd>The status code of the response, or the error code if the request could not be completed.</dd>
+    </dl>
+
+</script>
+
+<script type="text/javascript">
+    RED.nodes.registerType('ftp',{
+        category: 'output',
+        color:"rgb(231, 231, 174)",
+        defaults: {
+            name: {value:""},
+            method:{ value: "PUT" },
+            url: { value: "", validate: function (v) { return (v.trim().length === 0) || (v.indexOf("://") === -1) || (v.trim().indexOf("ftp") === 0) } },
+            username: { value: "dojot" },
+            password: { value: "dojot" },
+            filename: { value: "filename.txt" },
+            filecontent: { value: ""},
+            response: { value: { } }
+        },
+        inputs:1,
+        outputs:1,
+        icon: "white-globe.png",
+        label: function() {
+            return this.name||RED._("dojot/ftp:title");
+        },
+        labelStyle: function() {
+            return this.name?"node_label_italic":"";
+        },
+        paletteLabel:  RED._('dojot/ftp:title'),
+        oneditprepare: function() {
+            $("#node-input-filename").typedInput({ default: "msg", types: ["msg"] } );
+            $("#node-input-filecontent").typedInput({default:'msg',types:['msg']});
+            $("#node-input-response").typedInput({default:'msg',types:['msg']});
+        },
+        oneditsave: function() {
+            // if (!$("#node-input-usetls").is(':checked')) {
```

Comment:
@giovannicuriel 
I think Maybe You forgot  to remove ... rs

Reply:
I removed from http node. My bad.